lldb/packages/Python/lldbsuite/test/lldbutil.py

```diff
@@ -759,12 +760,17 @@ def run_to_breakpoint_make_target(test, exe_name = "a.out", in_cwd = True):
     # Create the target
     target = test.dbg.CreateTarget(exe)
     test.assertTrue(target, "Target: %s is not valid."%(exe_name))
+
+    # Set environment variables for the inferior.
+    if lldbtest_config.inferior_env:
+        test.runCmd('settings set target.env-vars {}'.format(lldbtest_config.inferior_env))
+
     return target
 
 def run_to_breakpoint_do_run(test, target, bkpt, launch_info = None):
     # Launch the process, and do not stop at the entry point.
     if not launch_info:
-        launch_info = lldb.SBLaunchInfo(None)
+        launch_info = target.GetLaunchInfo()
```

Comment:
Why did you change this call?

Reply:
To avoid reparsing the `lldbtest_config.inferior_env` string and populating environment vars into a fresh launch info.

Reply:
Makes sense.